qiskit/extensions/standard/multi_control_rotation_gates.py

```diff
@@ -35,14 +35,23 @@ def _apply_cu3(circuit, theta, phi, lam, control, target, use_basis_gates=True):
         circuit.cu3(theta, phi, lam, control, target)
 
 
+def _generate_gray_code(num_bits):
+    if num_bits <= 0:
+        raise ValueError(
+            "Must have at least 1 value to generate the gray code")
```

Comment:
Reword? Error sounds like there are not enough values.

Reply:
Updated in: https://github.com/Qiskit/qiskit-terra/pull/3949/commits/5d17db1ed7dd053a0583f73a70c2f6b1700ba826